<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtShanoirUploadWidgetDatasettypeDetailsSpectDataset</class>
 <widget class="QWidget" name="QtShanoirUploadWidgetDatasettypeDetailsSpectDataset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QTableWidget" name="processedDatasetTypeTable"/>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="spectDatasetNatureLabel">
         <property name="text">
          <string>Spect Dataset Nature :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="commentLabel">
         <property name="text">
          <string>Comment :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="commentLineEdit"/>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="spectDatasetNatureComboBox"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="processedDatasetTypeLabel">
         <property name="text">
          <string>Processed Dataset Type :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="processedDatasetTypeComboBox"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="checkAllButton">
         <property name="text">
          <string>Check All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="uncheckAllButton">
         <property name="text">
          <string>Uncheck All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
